doc/sphinx/source/modules/modules.rst

```diff
@@ -6,6 +6,10 @@ Standard Modules
 Standard modules are those which describe features that are considered
 part of the Chapel Standard Library.
 
+The modules :chpl:mod:`Assert` , :chpl:mod:`Math` , and :chpl:mod:`Types`
+are included by default.
```

Comment:
Minor quibble, but "included" is a vague term in this context.  'use'd might be clearer (with proper formatting).  "are implicitly 'use'd by every program by default" might be even better.